<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberInfoChangeWidget</class>
 <widget class="QWidget" name="MemberInfoChangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>会员信息修改</string>
  </property>
  <widget class="QLineEdit" name="Name_lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>姓  名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Agenda_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>性  别：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Agenda_comboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Birthday_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>生  日：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="Birthday_dateEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Tel_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>电  话：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Tel_lineEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Address_label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>地  址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Address_lineEdit">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>210</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>50</y>
     <width>187</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="Confirm_pushButton">
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Cancle_pushButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
